android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateManager.java

```diff
@@ -244,7 +245,16 @@ public void downloadPackage(JSONObject updatePackage, String expectedBundleFileN
                 }
 
                 if (isDiffUpdate) {
-                    CodePushUpdateUtils.verifyHashForDiffUpdate(newUpdateFolderPath, newUpdateHash);
+                    CodePushUtils.log("Applying diff update.");
```

Comment:
@max-mironov no, we can't, because logging should be executed anyway regardless signature verification enabled or not.